factory/django.py

```diff
@@ -172,12 +172,7 @@ def _get_or_create(cls, model_class, *args, **kwargs):
                 try:
                     instance = manager.get(**get_or_create_params)
                 except manager.model.DoesNotExist:
-                    raise ValueError(
-                        "django_get_or_create - Unable to create a new object "
-                        "due an IntegrityError raised based on "
-                        "your model's uniqueness constraints. "
-                        "DoesNotExist: Unable to find an existing object based on "
-                        "the fields specified in your factory instance.")
+                    raise e
```

Comment:
This can be a little confusing for someone seeing the code for the first time. Would you mind adding a small comment explaining that we are re-raising the original IntegrityError to align behavior with Django?